packages/api-explorer/src/reducers/spec/utils.spec.ts

```diff
@@ -23,20 +23,19 @@
  SOFTWARE.
 
  */
-import { ApiModel, SpecList } from '@looker/sdk-codegen'
+import { SpecList } from '@looker/sdk-codegen'
 import omit from 'lodash/omit'
 
 import { specs } from '../../test-data'
 import {
   getDefaultSpecKey,
-  parseSpec,
-  fetchSpec,
+  // parseSpec,
+  // fetchSpec,
```

Comment:
remove commented out code

Reply:
Ok. Not sure this was me, but I'll remove the cruft